app/models/report.rb

```diff
@@ -223,7 +223,7 @@ def data_student_contributions
     row_data = {}
     terms = Thesis.all.pluck(:grad_date).uniq.sort
     terms.each do |term|
-      row_data[term] = Thesis.where('grad_date = ?', term).map(&:student_contributed?).count(true)
+      row_data[term] = Thesis.where('grad_date = ?', term).includes(:versions).map(&:student_contributed?).count(true)
```

Comment:
Is the goal here to reduce the number of queries by loading only theses with papertrail versions?

Reply:
Nope. This eager loads the versions so the views don't get into an N+1 situation. Theses without versions should still load I believe (although I'm not sure if that's even a thing)